### ISSUE TITLE
DAMSL tags don't match

### ISSUE THREAD

**Issue by lzfelix** [cgpotts/swda#1]
Initially I would like to thank you for making this code available.

On section 1c of the coder's manual we can see the table with the 42 clustered labels, although it has 43 rows, as you mention on your page. However, one of these classes is _"% -"_, which can't be found on the dataset (I've performed a scan on it, and 0 matches were found). If the classes _"% -"_ and _"%"_ are merged (since both have a similar meaning), we are back to 42 classes as desired. This seemed to be done on Stolcke et al. [1] paper, as shown on Table 2. I've also noticed that on your page, the  _"% -"_ has the same full count as _"%"_.

[1] `Stolcke, Andreas, et al. "Dialogue act modeling for automatic tagging and recognition of conversational speech." Computational linguistics 26.3 (2000): 339-373. `

Thanks.

**Comment by ruizheliUOA** [comment]
@lzfelix You are right. When I processed this dataset, I also did not find any "% -" tag in that dataset. Meanwhile, I did not find how to process "+" tag in the dataset. Because there is no "+" tag in those 42 tags, but the number of "+" tag is over 10,000 in the original dataset. Whether the "+" tag is replaced with the corresponding tag of previous utterance from the same speaker?

**Comment by tnlin** [comment]
@ruizheliUOA same here, don't know what to do with "+" tag. Having check many paper but get no idea... replacing with the corresponding tag of previous utterance from the same speaker seems reasonable.

Reference:
1997_Switchboard SWBD-DAMSL Shallow-Discourse-Function Annotation
2000_Dialogue Act Modeling for Automatic Tagging and Recognition of Conversational Speech
2017_ Unsupervised Dialogue Act Induction using Gaussian Mixtures

**Comment by lzfelix** [comment]
To my understanding, you can either do that or simply disregard these utterances, depending on your problem.

**Comment by tnlin** [comment]
FYI, This paper mention about label "+" (finally...)
AAAI 2005 Dialogue Act Classification Based on Intra-Utterance Features
http://staffwww.dcs.shef.ac.uk/people/Y.Wilks/papers/AAAI05_A.pdf

**Comment by Shrinidhi-C** [comment]
Utterances marked as + are interrupted conversations 
These should be concatenated with continued dialogue of interrupted one
